package-lock.json

```diff
@@ -20,10 +20,10 @@
       "dev": true,
       "requires": {
         "@babel/types": "7.0.0-beta.44",
-        "jsesc": "^2.5.1",
-        "lodash": "^4.2.0",
-        "source-map": "^0.5.0",
-        "trim-right": "^1.0.1"
+        "jsesc": "2.5.1",
+        "lodash": "4.17.10",
+        "source-map": "0.5.7",
+        "trim-right": "1.0.1"
```

Comment:
I keep seeing PRs fighting carets between no carets. Is it because we're using different versions of npm?

Reply:
honestly, I have no idea. It might be because of npm version, and npm setting, or the way that individual dependencies declare _their_ dependencies ¯\\\_(ツ)_/¯